go-controller/pkg/ovn/egressip.go

```diff
@@ -1875,9 +1967,10 @@ func (e *egressIPZoneController) deleteReroutePolicyOps(ops []ovsdb.Operation, p
 // just remove the gatewayRouterIP from the list of nexthops
 // It also returns the list of podIPs whose routes and SNAT's were deleted
 // This function should be called with a lock on e.nodeZoneState.status.Node
-func (e *egressIPZoneController) deleteEgressIPStatusSetup(name string, status egressipv1.EgressIPStatusItem) ([]net.IP, error) {
+func (e *EgressIPController) deleteEgressIPStatusSetup(name string, status egressipv1.EgressIPStatusItem) ([]net.IP, error) {
 	var err error
 	var ops []ovsdb.Operation
+	defaultNetwork := util.DefaultNetInfo{}
```

Comment:
I see this is repeating everywhere.... I think its perhaps better to store a netInfo object in the EIP controller? or is this changing in future commits..?

Reply:
I dont think it would increase readability and it seems silly to just add that struct to the eip controller struct when i can just instantiate it. Its basically used as a constant.... so i guess i could make it a constant? It would make things slightly less readable but concise. Up to you..